src/test-gui/java/de/qabel/desktop/ui/AbstractPage.java

```diff
@@ -90,6 +88,11 @@ protected void fakeClick(String query) {
                 node.buildEventDispatchChain(new EventDispatchChainImpl()).dispatchEvent(event);
             }
         });
+
+        if (fireSufficesClick(node)) {
```

Comment:
now, two thing happen: the custom event is pushed to all handlers AND the fire method is called. previously, only one of them was called so you should better fix the `fireSufficesClick` method ot the fire call or remove the block ... but don't let 2 things get executed.